Automattic/vip
# Auto-login support

## Description

We make the link on the info table for dev-env to show admin link that is autologin. No need to enter username/password.
This changes the Login URL output on new or newly updated environments from:

```                                                 
 LOGIN URL         http://test.vipdev.lndo.site/wp-admin/
```

to:

```                                                                                       
 LOGIN URL         http://test.vipdev.lndo.site/wp-admin/?vip-dev-autologin=1111bbf8-284b-42b1-9a98-aa726d994f1d 
```

Also does all the configuration on update/create to set up the autologin token.
* Generate uuid token on every update/create
* Sets the token to environment json
* in lando file template uses this token and injects it as environment variable
* exposes the token in infor table login link


This is a sibling PR to https://github.com/Automattic/vip-container-images/pull/454

## Steps to Test

1) create/update environment
2) Start it and click the login URL
3) Should be logged in without username/password
